python/sauron-core/tests/test_stats.py

```
@@ -0,0 +1,162 @@
+    assert 'ai_score' in metrics
+    assert 0 <= metrics['ai_score'] <= 1
+
+def test_analyze_region_patterns():
+    """Test region pattern analysis"""
+    analyzer = StatisticalAnalyzer()
+
+    # Create test data
+    df = pd.DataFrame({
+        'region_entropy_var': [0.3, 0.4, 0.5],
+        'region_mean_var': [0.2, 0.3, 0.4],
+        'region_std_var': [0.1, 0.2, 0.3]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)